lib/ansible/runner.rb

```diff
@@ -58,14 +106,15 @@ def assert_hash!(hash)
       end
 
       def assert_path!(path)
+        return unless path
+
         unless File.exist?(path)
           raise "File doesn't exist: #{path}"
         end
       end
 
       def ansible_command(base_dir)
-        # TODO add possibility to use custom path, e.g. from virtualenv
-        "ansible-runner run #{base_dir} --json -i result"
+        "ansible-runner run #{base_dir} --json -i result --hosts localhost"
```

Comment:
Future stuff but we might want to make the `--hosts` configurable if people want to run playbooks on other systems i.e. post-provisioning